### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by anatolii-zarytskyi**

### CODE REVIEW

File: src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
```
Comment (KononenkoKonstantin) [inline src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java:12]: The private method should be after public

File: src/main/java/mate/academy/hibernate/relations/model/Actor.java [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+
```
Comment (KononenkoKonstantin) [inline src/main/java/mate/academy/hibernate/relations/model/Actor.java:17]: remove redundant empty line

File: src/main/java/mate/academy/hibernate/relations/model/Country.java [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
```
Comment (KononenkoKonstantin) [inline src/main/java/mate/academy/hibernate/relations/model/Country.java:15]: same

File: src/main/java/mate/academy/hibernate/relations/model/Movie.java [hunk 4]
```diff
@@ -2,10 +2,28 @@
...
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+
```
Comment (KononenkoKonstantin) [inline src/main/java/mate/academy/hibernate/relations/model/Movie.java:22]: same

**Review by KononenkoKonstantin — APPROVED** [review]

**Review by Max-Pochepets — APPROVED** [review]

**Pull request closed without merging**